workflow/kfdrc-somatic-variant-workflow.cwl

```diff
@@ -936,18 +936,22 @@ hints:
 - CNVKIT
 - CONTROLFREEC
 - CRAM
+- FUNCOTATOR
+- GATK
 - INDEL
 - LANCET
 - MAF
 - MANTA
 - MUTECT2
+- PON
```

Comment:
Not sure you need this tag as I think this WF does not actually run PON, it just exists within the repo overall

Reply:
The panel is an input.

Reply:
Ok removed.